spark3-extensions/src/main/antlr/org.apache.spark.sql.catalyst.parser.extensions/IcebergSqlExtensions.g4

```diff
@@ -73,6 +73,7 @@ statement
     | ALTER TABLE multipartIdentifier WRITE writeSpec                                       #setWriteDistributionAndOrdering
     | ALTER TABLE multipartIdentifier SET IDENTIFIER_KW FIELDS fieldList                    #setIdentifierFields
     | ALTER TABLE multipartIdentifier DROP IDENTIFIER_KW FIELDS fieldList                   #dropIdentifierFields
+    | ALTER TABLE multipartIdentifier ENCRYPTED WITH (KEK | MEK) BY TABLE KEY keyId         #createTableEncKey
```

Comment:
1. Are we going to add `BY COLUMN` later to support column level encryption? 
2. Maybe we don't need "BY TABLE", the key applies to all columns by default.

Reply:
Is there a plan to add this to command `CREATE TABLE`?

Reply:
Yes, I think we will need to support `CREATE TABLE` too, and also `BY COLUMN`, but `BY COLUMN` is for phase 2B.
I agree that `BY TABLE` seems not required. Removed for now.

Reply:
+1 on both accounts - adding this to `CREATE TABLE`, and replacing `BY TABLE KEY` with `BY KEY`